<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeDetailWindow</class>
 <widget class="QWidget" name="RecipeDetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="recipeNameLabel"/>
   </item>
   <item>
    <widget class="QLineEdit" name="recipeNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="recipeImageLabel"/>
   </item>
   <item>
    <widget class="QTextBrowser" name="recipeDescriptionBrowser"/>
   </item>
   <item>
    <widget class="QTextBrowser" name="recipeTextBrowser"/>
   </item>
   <item>
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
